servlet.py

```diff
@@ -696,8 +696,7 @@ def sanity_check():
     u8 = re.compile("UTF-?8", re.IGNORECASE)
     if not any(re.search(u8, os.environ.get(key, ""))
                for key in locale_vars):
-        print("servlet.py: error: APY needs a UTF-8 locale, please set LANG or LC_ALL",
-              file=sys.stderr)
+        print >>sys.stderr, "servlet.py: error: APY needs a UTF-8 locale, please set LANG or LC_ALL"
         sys.exit(1)
```

Comment:
$ python3
    Python 3.4.3+ (default, Oct 14 2015, 16:03:50) 
    [GCC 5.2.1 20151010] on linux
    Type "help", "copyright", "credits" or "license" for more information.
    >>> import sys
    >>> print >>sys.stderr, "servlet.py: error: APY needs a UTF-8 locale, please set LANG or LC_ALL"
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    TypeError: unsupported operand type(s) for >>: 'builtin_function_or_method' and '_io.TextIOWrapper'
    >>>